Auth0/CredentialsManager.swift

```diff
@@ -167,6 +167,41 @@ public struct CredentialsManager {
         self.retrieveCredentials(withScope: scope, callback: callback)
     }
     #endif
+    
+    /// Renew credentials regardless of expiry time. This will yield new credentials using the refreshToken regardless of whether the accessToken has expired.
+    /// Renewed credentials will be stored in the keychain.
+    ///
+    ///
+    /// ```
+    /// credentialsManager.renew(refreshToken: "some_refresh_token") { error, credential in
+    ///    guard error == nil else { return }
+    ///    print(credential)
+    /// }
+    /// ```
+    ///
+    /// - Parameters:
+    ///   - refreshToken: refreshToken to request new credentials.
```

Comment:
What is the purpose of passing in a refresh token? If you're going outside of the credentials manager I would suggest simply calling renew directly.

https://github.com/auth0/Auth0.swift/blob/master/Auth0/Auth0Authentication.swift#L338